LeaderboardBackend.Test/Controllers/LeaderboardsControllerTest.cs

```diff
@@ -56,17 +54,18 @@ public async Task GetLeaderboard_Ok_LeaderboardExists()
 	[Test]
 	public async Task GetLeaderboards_Ok_ListExists()
 	{
-		List<Leaderboard> mockList = new List<Leaderboard> {
-			new Leaderboard { Id = 1 },
-			new Leaderboard { Id = 2 },
+		var mockList = new List<Leaderboard>
+		{
+			new() { Id = 1 },
+			new() { Id = 2 },
 		};
 
 		_leaderboardServiceMock
-			.Setup(x => x.GetLeaderboards(It.IsAny<long[]>()))
-			.Returns(Task.FromResult<List<Leaderboard>>(mockList));
+			.Setup(x => x.GetLeaderboards(It.IsAny<ulong[]>()))
+			.Returns(Task.FromResult(mockList));
 
-		ActionResult<List<Leaderboard>> response = await _controller.GetLeaderboards(new long[] {1, 2});
+		var response = await _controller.GetLeaderboards(new ulong[] { 1, 2 });
 
-		Assert.AreEqual(new long[] {1,2}, response.Value?.ConvertAll(l => l.Id));
+		Assert.AreEqual(new long[] { 1, 2 }, response.Value?.ConvertAll(l => l.Id));
```

Comment:
Should the expected value here be a `ulong` array? I'm surprised this test passed

Reply:
Addressed in 76b68bfb5c7596c0a5c7d26ac909dfea1e14d13d.